js/about/preferences.js

```diff
@@ -654,8 +654,13 @@ class SecurityTab extends ImmutableComponent {
 
 class AdvancedTab extends ImmutableComponent {
   render () {
-    const pocketL10nArgs = {
-      extensionName: 'Pocket'
+    const extensionL10nArgs = {
+      pocket: {
+        extensionName: 'Pocket'
+      },
+      honey: {
+        extensionName: 'Honey'
+      }
     }
```

Comment:
@jonathansampson while that is pretty awesome (I love reduce!), I think the above is more readable